src/sdg/conf/base.py

```diff
@@ -574,3 +574,6 @@
     )
 
 SPECTACULAR_SETTINGS["SERVERS"] = SDG_API_SERVER_INSTANCES
+
+WHITELISTING_ENABLED = config("WHITELISTING_ENABLED", default=True)
```

Comment:
Dit is een SDG specifieke optie dus graag `SDG_` ervoor

```suggestion
SDG_API_WHITELISTING_ENABLED = config("WHITELISTING_ENABLED", default=True)
```